src/main/resources/data-config-sample.json

```diff
@@ -1,5 +1,6 @@
 {
     "project-settings": {
+        "year": "current",
```

Comment:
Reminder to update this to `copyright-year`

Reply:
Yeah, I will change this update over to what @Derthenier contributed.